_posts/2019-10-11-nottingham-tech-community.md

```diff
@@ -0,0 +1,43 @@
+
+![Statue of Robin Hood - Image by Ian Wilson from Pixabay](/assets/img/2019-10-11/robin-hood.jpg)
+
+## The growth of Tech Nottingham
+
+My first experience of tech meetups was back in 2010 when GeekUp Nottingham was born. In fact, my first experience of speaking in Nottingham was in [March 2011](http://geekup.org/events/283/). From that small seed, where 6 to 8 people would gather together, the [event grew into Tech Nottingham](https://www.technottingham.com/news/2014/4/25/geekup-nottingham-becomes-tech-nottingham) in 2014.
+
+Whilst Nottingham today has a rich variety of meetups, I don't personally believe the city's tech scene would be as strong as it is without [Tech Nottingham](https://www.technottingham.com/about) and the tireless work of Andrew & Emma Seward and the rest of the team. The core mission to _"Make Nottingham a better place to live and work in technology"_ has no greater legacy than the support and growth of meetups and speakers that have directly and indirectly benefitted from Tech Nottingham.
+
+A quick look at the [Nottingham.digital](https://nottingham.digital) website which lists most digital meetups in Nottingham shows 33 events currently listed; from tech, design, and ops topics. All free to attend in a wide range of locations with businesses and sponsors helping to make these events possible.
+
+## A seat of learning
+
+Nottingham is blessed with two world-class universities ( [Nottingham University](https://www.nottingham.ac.uk) and [Nottingham Trent University](https://www.ntu.ac.uk) ) and colleges. The benefit of strong ties into these institutions means more students are encouraged to take placements and find jobs in the local area.
+
+Meetups offer a change for students to get a better idea of what opportunities or topics are available around Nottingham and challenges the oft-held assumption that London is the next logical step for tech industry opportunities. The support flows into universities themselves with business and individuals volunteering their time to judge competitions, give guest lectures on business topics, or support the growing tech focussed student societies on campuses.
+
+The [Nottingham Growth Plan](https://nottinghaminsight.org.uk/themes/economy-and-development/strategies-and-plans/) has several strategies defined including 'Developing a skilled workforce'. 
+
+## The next evolution
+
+From small acorns to [Major Oak](https://www.visit-nottinghamshire.co.uk/things-to-do/the-major-oak-p586841)s meetup attendances are increasing in Nottingham. The next evolution of learning and sharing comes in the form of conferences like DDD East Midlands and [New Adventures](https://newadventuresconf.com/2020/).
```

Comment:
We've been using HTML for links so that they open in new tabs, but it's entirely up to you if that's something you want <a href="http://example.com/" target="_blank">example</a>